docs/run-platform-server/prerequisites.mdx

```diff
@@ -3,56 +3,41 @@ title: Prerequisites
 sidebar_position: 10
 ---
 
-Horizon only has one true dependency: a PostgreSQL server that it uses to store data that has been processed and ingested from Stellar Core. **Horizon requires PostgreSQL version >= 9.5**.
+The Horizon service is responsible for synchronizing with the Stellar network and processing ledger data. It performs the following functions:
```

Comment:
```suggestion
The Horizon service is responsible for synchronizing with the Stellar network and processing ledger data. We recommend reading the [Running section](./running.mdx) as a primer on scope of Horizon's services before diving into compute pre-requisites:
```

and for consideration, remove the 3 bullets for ingest, tx sub, api on lines 8 - 10 as the suggested link to Running covers that.